x-pack/platform/plugins/shared/lens/public/visualizations/tagcloud/tagcloud_toolbar/flyout_toolbar.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import type { ContentMap } from '../../../shared_components/flyout_toolbar';
+import { FlyoutToolbar } from '../../../shared_components/flyout_toolbar';
+import type { VisualizationToolbarProps } from '../../../types';
+import type { TagcloudState } from '../types';
+import { TagcloudAppearanceSettings } from './appearance_settings';
+
+export function TagcloudFlyoutToolbar(props: VisualizationToolbarProps<TagcloudState>) {
+  const flyoutToolbarContentMap: ContentMap<TagcloudState> = {
+    style: TagcloudAppearanceSettings,
+  };
+  return <FlyoutToolbar {...props} contentMap={flyoutToolbarContentMap} />;
+}
```

Comment:
This is a common pattern across all charts. Instead of having yet another function to call when we need to render the `FlyoutToolbarComponent` why not just passing this logic in `FlyoutToolbarComponent` directly (in ..._visualization.tsx file)
something like:
```
FlyoutToolbarComponent(props) {
  return <FlyoutToolbar {...props} contentMap={{style: TagcloudAppearanceSettings}} />;
},
  ```
we can save 1 file, 1 function call and some more naming